src/main/java/com/appdirect/sdk/meteredusage/config/MeteredUsageRetrofitConfiguration.java

```diff
@@ -0,0 +1,31 @@
+package com.appdirect.sdk.meteredusage.config;
+
+import lombok.RequiredArgsConstructor;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import retrofit2.Retrofit;
+import retrofit2.converter.jackson.JacksonConverterFactory;
+
+@Configuration
+@RequiredArgsConstructor
+public class MeteredUsageRetrofitConfiguration {
+
+	@Value("${appdirect.baseUrl:https://metered-usage}")
```

Comment:
Hmmm can we have a different name? It's not the `appdirect.baseUrl` actually, it's the one from meteredUsageService, what about?
```suggestion
	@Value("${meteredUsageService.baseUrl:https://metered-usage}")
```